cmd/explore.go

```diff
@@ -0,0 +1,165 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"atomicgo.dev/keyboard"
+	"atomicgo.dev/keyboard/keys"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/overmindtech/aws-source/proc"
+	"github.com/overmindtech/cli/tfutils"
+	"github.com/overmindtech/discovery"
+	"github.com/overmindtech/pterm"
+	stdlibsource "github.com/overmindtech/stdlib-source/sources"
+	log "github.com/sirupsen/logrus"
+	"github.com/sourcegraph/conc/pool"
+	"github.com/spf13/cobra"
+	"golang.org/x/oauth2"
+)
+
+// exploreCmd represents the explore command
+var exploreCmd = &cobra.Command{
+	Use:    "explore",
+	Short:  "Run local sources for using in the Explore page",
```

Comment:
I don't think it should block the merge but I'd like @jameslaneovermind to look at the descriptions here and jazz them up a bit